djorg/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for djorg project.
+
+Generated by 'django-admin startproject' using Django 2.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '8$g!m2+wu83mm%c5ygw@137469+0@s=p*3ps5(v^alzd!4n(m8'
```

Comment:
Remember to conceal your SECRET_KEY in a .env file before pushing to deployment. Having this exposed is a matter of security, although it isn't the end of the world until you have sensitive information that can be accessed.

Reply:
Please reach out to me if you are not familiar with .env files. This is very helpful as you move from development to production builds.